test/unit/org/apache/cassandra/dht/BootStrapperTest.java

```diff
@@ -68,22 +63,6 @@
 public class BootStrapperTest extends CassandraTestBase
 {
     static Predicate<Replica> originalAlivePredicate = RangeStreamer.ALIVE_PREDICATE;
-    public static AtomicBoolean nonOptimizationHit = new AtomicBoolean(false);
```

Comment:
trunk patch conflicted with our changes, and just using the trunk version passes...

Reply:
on cep-15-accord this was rewritten in

```
commit dec690f93a422d8b2aa9aa622585965807fac3fb
Author: Ariel Weisberg <aweisberg@apple.com>
Date:   Mon Jan 23 14:58:21 2023 -0500

    Accord/non-Accord interoperability and support for live migration

    Patch by Ariel Weisberg; Reviewed by Blake Eggleston for CASSANDRA-18129

    Co-authored-by: Blake Eggleston <beggleston@apple.com>
```

The trunk conflict was

```
commit fcea0b6fd8aa6685e7a7e4be5bcde0ee87efc75f
Author: Bereng <berenguerblasi@gmail.com>
Date:   Thu Apr 10 13:33:33 2025 +0200

    CASSANDRA-19633 Replaced node is stuck in a loop calculating ranges
```

Reply:
ok trunk version is passing, going to push a patch now